pr_watch/github.py

```diff
@@ -143,14 +145,21 @@ def get_pr_list_from_username(user_name, fork_name):
     """
     Retrieve the current active PRs for a given user
     """
-    q = 'is:open is:pr author:{author} repo:{repo}'.format(author=user_name, repo=fork_name)
+    return get_pr_list_from_usernames([user_name], fork_name)
+
+
+def get_pr_list_from_usernames(user_names, fork_name):
+    """
+    Retrieve the current active PRs for a given set of users
+    """
+    authors = ' '.join('author:{author}'.format(author=author) for author in user_names)
+    q = 'is:open is:pr {authors} repo:{repo}'.format(authors=authors, repo=fork_name)
     r_pr_list = get_object_from_url('https://api.github.com/search/issues?sort=created&q={}'.format(q))
-    logger.debug('List of PRs received for user %s: %s', user_name, r_pr_list)
 
     pr_list = []
     for pr_dict in r_pr_list['items']:
-        pr = get_pr_by_number(fork_name, pr_dict['number'])
-        pr_list.append(pr)
+        logger.debug('Received PR for user %s: %s', pr_dict['user']['login'], pr_dict)
+        pr_list.append(get_pr_by_number(fork_name, pr_dict['number']))
```

Comment:
It took me a while to understand why we need to re-request each PR, since we already got information about each PR in the search results.  The information we get from the search result is not detailed enough, though, so we have to query each individual PR.